docs/data/material/customization/theming/theming.md

```diff
@@ -77,7 +77,18 @@ The community has built great tools to build a theme:
 
 ## Accessing the theme in a component
 
-You [can access](/system/styles/advanced/#accessing-the-theme-in-a-component) the theme variables inside your React components.
+You [can access](/system/styles/advanced/#accessing-the-theme-in-a-component) the theme variables inside your functional React components using the `useTheme` hook:
```

Comment:
```suggestion
You can access the theme variables inside your functional React components using the `useTheme` hook:
```

The reference to the legacy pages was creating the confusion, I would just remove it.